### PR TITLE
add computedrole column for DPUB mappings

### LINKED ISSUE #29: Define computedrole for doc-* roles
Based on the outcome of this ARIA issue…
- https://github.com/w3c/aria/issues/1887

DPUB-AAM (or dpub-aria) will likely need an update similar to:
- https://github.com/w3c/core-aam/pull/167

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes #29
- #29

Prerequisite for:
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/71
- https://github.com/web-platform-tests/wpt/pull/42986

Core-AAM counterpart PR was: 
- https://github.com/w3c/core-aam/pull/167

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/dpub-aam/pull/33.html" title="Last updated on Nov 7, 2023, 4:25 PM UTC (1be2480)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/dpub-aam/33/a3412f8...1be2480.html" title="Last updated on Nov 7, 2023, 4:25 PM UTC (1be2480)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -305,6 +305,10 @@ <h4 id=role-map-abstract><code>doc-abstract</code></h4>
         <a class="dpub-role-reference" href="#doc-abstract"><code>doc-abstract</code></a>
       </td>
     </tr>
+    <tr>
+      <th><a class="core-mapping" data-cite="core-aam#roleMappingComputedRole">Computed Role</a></th>
```
Comment (cookiecrook) [inline index.html:309]: @spectranaut This link references Core-AAM 1.2, which is not yet live. What should it be instead? ED link?

Comment (cookiecrook) [reply index.html]: NM... My mistake. Should be resolved now.

**Review by spectranaut — APPROVED** [review]
This looks good to me! Maybe an editor should merge, just to see it, @TzviyaSiegman @mattgarrish  You can read about the purpose of this string here: https://w3c.github.io/core-aam/#roleMappingComputedRole

**Comment by mattgarrish** [comment]
Thanks for the info. Looks good to me.

**Comment by cookiecrook** [comment]
Thanks Matt and Valerie!

**Pull request merged**